### PR TITLE
code base

### PR DESCRIPTION
**Pull request by martinicolaevlad**
am facut file u MainPage super basic cu componentele pe care am povestit ca le vrem.

### CODE REVIEW

File: g9frontend/src/components/WordSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+//                     <div className="square"></div>
+//                 </div>
+//             </div>
+//             <footer className="footer">Centered text</footer>
+//         </div>
+//     )
+// }
+
+// export default WordSearch
+//
```
Comment (bia1708) [inline g9frontend/src/components/WordSearch.tsx:45]: Same question as above

File: g9frontend/src/components/MainPage.tsx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    //         <div className="left-container"></div>
+    //         <div className="right-container">
+    //             <div className="semicircle"></div>
+    //             <div className="square"></div>
+    //             <div className="square"></div>
+    //             <div className="square"></div>
+    //         </div>
+    //     </div>
+    //     <footer className="footer">Centered text</footer>
+    // </div>
```
[the remark this answers is not in the document]
Comment (martinicolaevlad) [reply g9frontend/src/components/MainPage.tsx:140]: it was a template for me

**Comment by Sipos-Lucas-George** [comment]
So just to be sure, there was nothing made in the code base, please, when you got time delete unnecessary files, such as favorite.ico, pngs etc. and make the basic routing, e.g.: "/" -> MainPage()

**Comment by bia1708** [comment]
@Sipos-Lucas-George there have been made changes to the default react project. @martinicolaevlad did a basic layout of the components. 
This is what it looks like: 
image_2023-11-03_184225413

**Comment by bia1708** [comment]
Also, @martinicolaevlad, when you find some time, please provide a basic README.md with the run commands for the app (i.e. `npm install` and `npm start` and whatever else is needed) and comment your code.

File: g9frontend/src/components/WordSearch.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+            backgroundColor: 'green',
+            marginTop: '8px',
+            padding: '8px',
+            display: 'flex',
+            justifyContent: 'space-between',
+            color:'white',
+        };
+        const [text, setText] = React.useState('');
+    return (
+        <div style={containerStyle} className="root-container">
```
Comment (Andrei-Fabian-Pop) [inline g9frontend/src/components/WordSearch.tsx:152]: You could add a folder called componentbase where you add a template for a plain component so when the tasks are assigned everyone has a starting component in which they  can solve their task. Each component should have a small size in terms of lines of code (you could specify this as a project rule). Making small components such as RootContainer that contain other components and so on will allow us to scale the project and ensure high cohesion and low coupling.

**Review by Andrei-Fabian-Pop — CHANGES_REQUESTED** [review]
Overall the project base looks good, i only had 2 questions. Apart from that I just want to mention that the react components should be small in terms of lines of code and many files, this way we will save time in the long run because everyone will work on their components and we'll be able to reuse most of them, not change the code written by someone else.

**Review by bia1708 — CHANGES_REQUESTED** [review]

**Review by bia1708 — CHANGES_REQUESTED** [review]
Before merging, you also need to solve the conflict with the README.md file.
Please sign your commits and address the requested changes. Before making the changes and signing the commits, please also consult the Contribution Rules on the organization's front page and rename the commits appropriately.
The `SearchBar` component is fine for now, but it is not finished. Later on, you will have to handle sending the searched word as a JSON to the backend.

**Review by bia1708 — APPROVED** [review]
Thank you for the code base, good job <3
From now on please keep your commits tidier, but that's about it.

**Review by Andrei-Fabian-Pop — APPROVED** [review]
Really cool stuff

**Comment by martinicolaevlad** [comment]
Ly man

**Pull request merged**